weave/index/index_sql.py

```diff
@@ -286,14 +287,20 @@ def to_pandas_df(self, max_rows=1000, offset=0, **kwargs):
             pantry.
         """
         # Get the rows from the index as a list of lists, then get the columns.
-        result, columns = self.execute_sql(
+        if max_rows > 0:
```

Comment:
I think it makes more sense here to do "if max_rows is not None" because it doesn't make sense to check max_rows against 0 since max_rows can be None.

Reply:
I've gone through and updated this and similar issues. Good catch.